src/main/java/jackyy/dimensionaledibles/registry/ModConfig.java

```diff
@@ -113,14 +140,20 @@
             public boolean useWorldSpawn = true;
```

Comment:
The comment text for this property should be changed from saying "Overworld Apple" to "Overworld Cake".